TSC-Charter.md

```diff
@@ -65,7 +65,6 @@ TSC members are expected to regularly participate in TSC activities.
 A TSC member is automatically removed from the TSC if, during a 3-month period,
 all of the following are true:
 
-* They attend fewer than 25% of the regularly scheduled meetings.
 * They do not participate in any TSC votes.
```

Comment:
Since we're reducing the list to one item, maybe inline it?


```suggestion
A TSC member is automatically removed from the TSC if they do not participate
in any TSC votes during a 3-month period wherein there is at least 1 TSC vote.
```

Reply:
I believe that wording would mean that anyone who didn't vote in the previous vote would be then removed from the TSC automatically as soon as this PR lands? I don't believe that's what you're intending.

Similarly, given infrequency of votes, is missing a single vote a reasonable grounds for someone being automatically removed?

Reply:
> Similarly, given infrequency of votes, is missing a single vote a reasonable grounds for someone being automatically removed?

I think it is reasonable. The TSC can vote people back on immediately if it is appropriate. Votes are kept open typically for a week or so. Sometimes, people will be on vacation or whatever, and that's fine. The TSC can act on those on a case-by-case basis. Being removed isn't punishment.